modules/openapi-generator/src/main/resources/typescript/generators/fetch-api.mustache

```diff
@@ -1,7 +0,0 @@
-export * from './http/http';
```

Comment:
Should `generators/jquery.mustache` also be removed?

Reply:
Ah yes definitely, guess that got lost during rebasing.

Reply:
Ok, removed it now.